### DISCUSSION TITLE
[Feature Request] Add sequential order

### DISCUSSION THREAD

**Discussion by Gh20221205** [adieyal/sd-dynamic-prompts#506] [Q&A] [answered]
Random order is great, but sometimes I just want to have some control over the order, for instance, I want to do a side by side comparison of several character loras. I have saved these loras in lora.txt, but since the row is randomly picked, sometimes I have trouble linking character to lora used. If there is an option of sequential order, it will be straightforward. Another scenario is that I want to have control over lora used for each face in a group picture.

So can you add an option to allow sequential order, something like,  `||lora||` will pick a row from the first to last in sequential order (feature requested) while `__lora__` will pick a random row from lora.txt (current feature)?

Thanks!

**Comment by akx** [comment, marked as answer]
Have you tried [combinatorial generation](https://github.com/adieyal/sd-dynamic-prompts#combinatorial-generation)?

**Reply by Gh20221205** [reply]
In a scenario that there are two faces in the image, and I want to replace each face with a different face from a different lora. I tried putting following in the prompt for Adetailer but it seems it doesn't work. Any idea?
`1girl, {<lora1>|<lora2>}`

**Comment by bobpuffer1** [comment]
I write my loras into my dynamic prompt like any other variant, set the weight appropriately and make it the last line of the prompt to get it to compare side-by-side.

**Reply by Gh20221205** [reply]
Have you tried to generate more than one characters in one image, and use a specific face to use a specific lora, say, face1 using lora1, face2 using lora2?
